other/materials_designer/specific_examples/grain_boundary_3d_fcc_metals.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p>Let's add copper into the title and filename (and mkdocs link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>